docs/reference/remap/get_env_var.cue

```diff
@@ -0,0 +1,31 @@
+package metadata
+
+remap: functions: get_env_var: {
+	arguments: [
+		{
+			name:        "name"
+			description: "Name of the environment variable."
+			required:    true
+			type: ["string"]
+		},
+	]
```

Comment:
Errors are now being documented in a new `internal_failure_reasons` section. So here it should say:

```
        internal_failure_reasons: [
                "`value` is an environment variable that does not exist",
        ]        
```

Reply:
:+1: though `value` should be `name` there.

Reply:
Added!